src/commands/reinvest.rs

```diff
@@ -1,18 +1,18 @@
-use std::{convert::TryFrom, ops::Add, path, sync::Arc, time::Duration};
+use std::{convert::TryFrom, path, sync::Arc, time::Duration};
+
+use abscissa_core::{Application, Clap, Command, Runnable};
 
-use abscissa_core::{Application, Command, Clap, Runnable};
-use chrono::Utc;
 use ethers::prelude::*;
-use num_bigint::{BigInt, ToBigInt};
-use num_traits::Zero;
+
```

Comment:
Here and elsewhere, we should probably delete unnecessary newlines between `use` statements.